src/visitpy/common/MethodDoc.C

```diff
@@ -8842,12 +8843,12 @@ const char *visit_SuppressMessages_doc =
 "\n"
 "Synopsis:\n"
 "\n"
-"SuppressMessages(int level) ->integer\n"
+"SuppressMessages(level) -> integer\n"
```

Comment:
Adding this note here because I couldn't attach to an earlier line.

Maybe add a line for `ToggleLockTools` up around line 8653 so that it too appears when someone types `help(ToggleLockTools)`

Reply:
I plan to make several tickets for updating documentation areas that are lacking. This is one of those areas. If the activities of ToggleLockTools were described in the description portion, then I would just add it to the list in the synopsis here because that is a very quick fix. However, ToggleLockTools is not mentioned in the description either. That wouldn't take long to do either, especially since you provided information on what it does in the ticket comments, but there quite a few of these, and I don't want to get side-tracked. Plus, once the reverse script is done, this provides some helpful tests for modifying existing functions, so I can do it then.

Reply:
Ok, good division of labor strategy. Yeah, with a lot of moving parts its easy to get distracted.